katharsis-jpa/src/main/java/io/katharsis/jpa/internal/query/AbstractQueryExecutorImpl.java

```diff
@@ -95,8 +95,7 @@ public MetaDataObject getMeta() {
 				entityList.add((T) values[0]);
```

Comment:
welcome to the enterprise community...

Lambdas and stuff are nice, but there are currently also plenty of other things to worry.